### Notebook Diffs vs origin/main
NOTEBOOK: nb1.ipynb

nbdiff /dev/fd/63 nb1.ipynb
--- /dev/fd/63  2025-07-30 14:03:49.341951
+++ nb1.ipynb  2025-07-30 14:03:37.032874
## inserted before /cells/3:
+  code cell:
+    source:
+      CLEAN UP 




